cli/main.c

```diff
@@ -408,9 +459,13 @@ static int status(int argc, char **argv)
 		struct switchtec_dev *dev;
 		int reset_bytes;
 		int verbose;
+		int fmt;
 	} cfg = {};
 	const struct argconfig_options opts[] = {
 		DEVICE_OPTION,
+		{"format", 'f', "FMT", CFG_CHOICES, &cfg.fmt, required_argument,
+		 "output format (default: normal)",
+		 .choices=output_fmt_choices},
 		{"reset", 'r', "", CFG_NONE, &cfg.reset_bytes, no_argument,
 		 "reset byte counters"},
 		{"verbose", 'v', "", CFG_NONE, &cfg.verbose, no_argument,
```

Comment:
If the verbose mode only work for 'normal' output, better we have this info in the 'verbose' help.